build/test/prepare-plugin.sh

```diff
@@ -14,27 +14,29 @@ else
 fi
 
 # Build
-cd .. && git clone -b ${GIT_BRANCH} https://github.com/mattermost/mattermost-plugin-calls --depth 1 && \
-cd mattermost-plugin-calls &&
+cd .. && git clone -b ${GIT_BRANCH} https://github.com/mattermost/mattermost-plugin-calls && \
+cd mattermost-plugin-calls && \
+git fetch --tags && \
 cd standalone && npm ci && cd .. && \
 cd webapp && npm ci && cd .. && \
-echo 'replace github.com/mattermost/rtcd => ../rtcd' >> go.mod && \ # We need to make sure we compile the plugin with the rtcd changes.
+echo "replace github.com/mattermost/rtcd => ../rtcd" >> go.mod && \
 go mod tidy && \
 make dist MM_SERVICESETTINGS_ENABLEDEVELOPER=true
 
 # Installation
 PLUGIN_BUILD_PATH=$(realpath dist/*.tar.gz)
 PLUGIN_FILE_NAME=$(basename ${PLUGIN_BUILD_PATH})
 
-docker cp ../rtcd/build/test/config_patch.json mmserver_server_1:/mattermost && \
-docker exec mmserver_server_1 bin/mmctl --local config patch config_patch.json && \
-docker cp ${PLUGIN_BUILD_PATH} mmserver_server_1:/mattermost && \
-docker exec mmserver_server_1 bin/mmctl --local plugin delete ${PLUGIN_ID} && \
-docker exec mmserver_server_1 bin/mmctl --local plugin add ${PLUGIN_FILE_NAME} && \
-docker exec mmserver_server_1 bin/mmctl --local plugin enable ${PLUGIN_ID} && \
+docker ps -a && \
```

Comment:
Is this for testing, or needed?

Reply:
Let's say I wish I had it before going through this. It doesn't hurt as it adds more visibility into the pipeline.